### PR TITLE
*: add "getting started" example for LDAP

### PR DESCRIPTION
**Pull request by ericchiang**
This is how I tested my changes for https://github.com/coreos/dex/pull/1021

Add a getting started guild using OpenLDAP. Hopefully others find this useful.

Not urgent.

cc @rithujohn191 @estroz

### CODE REVIEW

**Comment by ericchiang** [comment]
cc @dgonyeo you might be interested in the OpenLDAP standup script

File: Documentation/dev-integration-tests.md [hunk 1]
```diff
@@ -50,9 +50,15 @@ $ sudo ./storage/sql/standup.sh destroy postgres
 
 ## LDAP
 
-To run LDAP tests locally, you require a container running OpenLDAP.
+The LDAP integration tests require [OpenLDAP][openldap] installed on the host machine. To run them, use `go test`:
 
-Run OpenLDAP docker image:
+```
+DEX_LDAP_TESTS=1 go test -v ./connector/ldap/
```
Comment (rithujohn191) [inline Documentation/dev-integration-tests.md]: Can we change this to 
```
export DEX_LDAP_TESTS=1
go test -v ./connector/ldap/
```

File: Documentation/ldap-connector.md [hunk 2]
```diff
@@ -9,6 +9,30 @@ The connector executes two primary queries:
...
+./bin/dex serve examples/config-ldap.yaml
+```
+
+Then run the OAuth client in another terminal.
+
+```
+./bin/example-app
+```
+
+Go to [http://localhost:5555](http://localhost:5555), login and enter the username and password of the LDAP user: `janedoe@exapmle.com`/`foo`. Add the "groups" scope as part of the initial redirect to add group information from the LDAP server.
```
Comment (rithujohn191) [inline Documentation/ldap-connector.md]: janedoe@exapmle.com --> janedoe@example.com

File: examples/ldap/openldap.schema [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+## <http://www.OpenLDAP.org/license.html>.
+
+#
+# OpenLDAP Project's directory schema items
+#
+# depends upon:
+#	core.schema
+#	cosine.schema
+#	inetorgperson.schema
+#
```
Comment (rithujohn191) [inline examples/ldap/openldap.schema]: Are the schema files included to make it easier to use? Won't installing OpenLDAP fetch these?

Comment (ericchiang) [inline examples/ldap/openldap.schema]: Mentioned that here https://github.com/coreos/dex/pull/1022/commits/60637b51ca6ce7b3a5e6fc07545f80606d6531f0#diff-3936bdc133814c090b1aa375b6fe6717R25

Basically, I found it doesn't install them in predictable places. This is stolen from the current LDAP integration tests.

Comment (ericchiang) [inline examples/ldap/openldap.schema]: Actually, maybe we should just have the tests and spin up script reference the same schema files.

Comment (rithujohn191) [inline examples/ldap/openldap.schema]: If the schema files get installed in a fixed location thats platform independent then yea we could just have the script reference it.

Comment (ericchiang) [inline examples/ldap/openldap.schema]: I mean, these are currently copied from our integration tests. https://github.com/coreos/dex/tree/master/connector/ldap/testdata

We should probably only hold them in a single place.

**Comment by ericchiang** [comment]
comments addressed

**Pull request merged**